evalml/model_understanding/graphs.py

```diff
@@ -516,8 +541,50 @@ def graph_partial_dependence(pipeline, X, feature, class_label=None, grid_resolu
             msg = f"Class {class_label} is not one of the classes the pipeline was fit on: {', '.join(list(pipeline.classes_))}"
             raise ValueError(msg)
 
-    part_dep = partial_dependence(pipeline, X, feature=feature, grid_resolution=grid_resolution)
-    feature_name = str(feature)
+    part_dep = partial_dependence(pipeline, X, features=features, grid_resolution=grid_resolution)
+
+    if isinstance(features, (list, tuple)):
+        fig = graph_two_way_part_dep(_go, features, part_dep)
+    else:
+        fig = graph_one_way_part_dep(_go, class_label, features, part_dep, pipeline)
+
+    return fig
+
+
+def graph_two_way_part_dep(_go, features, part_dep):
```

Comment:
Are these just meant to be private helper functions for `graph_partial_dependence`? :O If yes, let's prefix with an underscore to indicate that, if no and they're meant to be user-facing, let's add them to the API ref!

Reply:
Just to resolve - this is now gone.  I've merged all back into the original graph_partial_dep().  But that's a good call.